templates/gitlab-ci-template-cloud-run.yml

```diff
@@ -43,16 +42,16 @@ stages:
 variables:
   CLOUDSDK_CORE_DISABLE_PROMPTS: 1 # Disable all interactive prompts for gcloud (https://cloud.google.com/sdk/gcloud/reference#--quiet)
   GCP_REGION: europe-west1
-  BRANCHES_MANUAL_DEPLOYMENT: "/^(develop)$/"
-  BRANCHES_AUTOMATIC_DEPLOYMENT: "/^(master|main)$/"
+  DEPLOYMENT_MAIN_BRANCH: main
+  DOCKER_IMAGE_NAME: ${APP_NAME}-${CI_COMMIT_REF_SLUG}
   CI_REGISTRY_IMAGE_BASE_URL: gcr.io/${GCP_PROJECT_ID}
   CI_REGISTRY_IMAGE_TAG: ${CI_COMMIT_SHORT_SHA}
-  CI_REGISTRY_IMAGE: ${CI_REGISTRY_IMAGE_BASE_URL}/${APP_NAME}:${CI_REGISTRY_IMAGE_TAG}
-  # DEPLOYMENT_SUFFIX:
-  # SERVICE_ACCOUNT:
+  CI_REGISTRY_IMAGE: ${CI_REGISTRY_IMAGE_BASE_URL}/${DOCKER_IMAGE_NAME}:${CI_REGISTRY_IMAGE_TAG}
   DEPLOYMENT_NAME: ${APP_NAME}-${CI_COMMIT_REF_SLUG}
   PORT: 80
   DOCKERFILE: Dockerfile
```

Comment:
i was thinking to enforce the usage of `DOCKER_BUILDKIT=1` as we do in other places

Reply:
Fixed